Sources/Core/JavaModelRenderer.swift

```diff
@@ -327,8 +332,8 @@ public struct JavaModelRenderer: JavaFileRenderer {
                 "java.util.Objects",
                 "java.lang.annotation.Retention",
                 "java.lang.annotation.RetentionPolicy",
-                "android.support.annotation.NonNull",
-                "android.support.annotation.Nullable",
+                nullabilityAnnotationType!.package + ".NonNull",
+                nullabilityAnnotationType!.package + ".Nullable",
```

Comment:
if you use `guard let` above, that should satisfy the compiler check for nullability and removes the need of this force cast